app/javascript/crayons/Buttons/__stories__/Buttons.mdx

```diff
@@ -14,35 +14,50 @@ That is: `<CTA />` or `<Link />` should be used where navigation is elicited, an
 For example, a 'Submit' control on 'Sign in' view (form) would suit a `<Button />` because its job is to submit the form. 
 However, 'Register' call-to-action—taking the user to a different form, in a different page or screen—would benefit from `<CTA />` or `<Link />` semantics.
 
-Do not use `<Button />s` as navigational elements. Instead, use Links (or CTAs) when the desired action is to take the user to a new page.
+Do not use `<Button />s` as navigational elements. Instead, use `<Link />s` (or `<CTA />s`) when the desired action is to take the user to a new page.
 
 ## Variants
 
+### Default
+
+_Default_ variant is not very prominent, and therefore can be used for less prominent actions.
```

Comment:
Wording (blocking): Incomplete sentence. Start with `The`. There are other instances of this below as well in the rest of the documentation.

```suggestion
The _Default_ variant is not prominent, and therefore can be used for less prominent actions.
```

Wording (nitpick):

```suggestion
The _Default_ variant is not prominent. Use it potentially for less noticeable actions.
```

Reply:
_The_ vs _A_ was always my weak point in English... :( I tried to update it few examples, but... 🤷‍♂️